src/c++/perf_analyzer/genai-perf/genai_perf/tokenizer.py

```diff
@@ -21,16 +21,19 @@
 with contextlib.redirect_stdout(io.StringIO()) as stdout, contextlib.redirect_stderr(
     io.StringIO()
 ) as stderr:
-    from transformers import AutoTokenizer
+    from transformers import AutoTokenizer, PreTrainedTokenizer, PreTrainedTokenizerFast
     from transformers import logging as token_logger
+    from transformers.tokenization_utils_base import BatchEncoding
 
     token_logger.set_verbosity_error()
 
-
+Tokenizer = PreTrainedTokenizer | PreTrainedTokenizerFast
```

Comment:
What does this do?
Is this setting a default type?

Reply:
This is aliasing Tokenizer to these two types, so that we do not need to repeat them everywhere we use them. This is what Autotokenizer.from_pretrained("tokenizer_name") returns.

Reply:
Gotcha. thank you